settings.gradle

```diff
@@ -58,4 +58,5 @@ include 'status-checker'
 include 'flow-api'
 include 'template-manager-tag'
 include 'common-handlebar'
+include 'environment-support'
```

Comment:
call it maybe `environment-common` or similar if we would like to put more env related stuff here. if it's just proxy related it should be reflected in its name